hx_lti_initializer/views.py

```diff
@@ -76,7 +77,13 @@ def initialize_lti_tool_provider(req):
 
 def create_new_user(username, user_id, roles):
     # now create the user and LTIProfile with the above information
-    user = User.objects.create_user(username, user_id)
+    # Max 30 length for person's name, do we want to change this? It's valid for HX but not ATG/FAS
```

Comment:
It turns out that the username field is actually from django auth, so we can't change the size or allow duplicates. Currently in the process of exploring the other options.